notebooks/dc2/catalog_generation/matchingLensGalaxies.ipynb

```diff
@@ -0,0 +1,6293 @@
+    "    for om10_row in om10_systems:\n",
+    "        log_z_om10 = np.log10(om10_row['ZSRC'])\n",
+    "        imag_om10 = om10_row['MAGI_IN']\n",
+    "        \n",
+    "        matches = np.where((np.abs(np.log10(redshift) - log_z_om10) < 0.03) &\n",
+    "                           (np.abs(agn_i_mag - imag_om10) < 0.05))[0]\n",
+    "        keep_matches = [w for w in matches if w not in host_gal_idx]\n",
+    "        if len(keep_matches) > 0:\n",
+    "            gal_match = rand_state.choice(keep_matches)\n",
+    "            om10_idx.append(i)\n",
+    "            host_gal_idx.append(gal_match)\n",
+    "            \n",
+    "        i += 1\n",
+    "        \n",
+    "    return om10_idx, host_gal_idx, om10_systems['LENSID'][om10_idx]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 107,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "om10_index, agn_host_gal_index, om10_matched_lensid = match_hosts_om10(full_agn_host_join['redshift_true'],\n",
+    "                                                                       full_agn_host_join['mag_i_agn'],\n",
+    "                                                                       om10_ddf_systems, seed=274)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 108,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(1007, 1200)"
+      ]
+     },
+     "execution_count": 108,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(om10_index), len(om10_ddf_systems)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 109,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(array([0.  , 0.05, 0.1 , 0.15, 0.2 , 0.25, 0.3 , 0.35, 0.4 ]),\n",
+       " <a list of 9 Text yticklabel objects>)"
+      ]
+     },
+     "execution_count": 109,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAzgAAAMuCAYAAADVA6YEAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzde5xVVd348c8XARVBTUEQfRAv4Q9vj8ZQIZoo4gUf81qaF8TsQUQxIzUvFWhGmeblwcjLI+Ili1IrTVHUx0teSkHNUDJN0OSOlqaICK7fH/tAM2fmMGeYy5nZfN6v13kxs/faa3/X3mfO4nvW3mtHSglJkiRJyoN2lQ5AkiRJkpqKCY4kSZKk3DDBkSRJkpQbJjiSJEmScsMER5IkSVJumOBIkiRJyo32lQ6gWNeuXVPv3r0rHYYkrfNmzJixJKXUrdJxtEb2VZJUeaX6qVaX4PTu3Zvp06dXOgxJWudFxBuVjqG1sq+SpMor1U95iZokSZKk3DDBkSRJkpQbJjiSJEmScsMER5IkSVJumOBIkiRJyo1WN4ua1Bzee+89Fi1axMcff1zpUKRWoUOHDmyxxRZsvPHGlQ5FEvZTUrHG9FMmOMq99957j4ULF7LVVlux4YYbEhGVDkmqqJQSH374IXPnzgUwyZEqzH5Kqqmx/ZSXqCn3Fi1axFZbbUWnTp3sNCQgIujUqRNbbbUVixYtqnQ40jrPfkqqqbH9lAmOcu/jjz9mww03rHQYUquz4YYbejmM1ArYT0l1W9t+ygRH6wS/EZNq8+9Caj38e5RqW9u/CxMcSZIkSblhgiOpVZk8eTKdO3cu+XtrFxHccccdja6nd+/eXH755U0QkSSpKdlPZVpzP+Usalpn9T7v3hbd35wfHtLgbebOnctFF13Efffdx6JFi+jWrRtDhw5l7NixbL311qvLDR8+nJtvvpmvfvWr
```

Comment:
We are missing the SIS-approximated Einstein radius and the lambda factor (which accounts for the ellipticity of projected mass), but these quantities can be computed from `vel_disp_lenscat` and `ellip_lenscat`, respectively, so maybe this is fine!

Reply:
Yeah, I changed these in a new version of the notebook. These values will have to come from DC2 since we can't change the lens galaxies that are already there. So, we'll have to calculate them from the vel_disp and ellipticity.